packages/create-svelte/templates/default/src/routes/index.svelte

```diff
@@ -14,8 +14,8 @@
 	<h1>
 		<div class="welcome">
 			<picture>
-				<source srcset="svelte-welcome.webp" type="image/webp">
-				<img src="svelte-welcome.png" alt="Welcome"/>
+				<source srcset="svelte-welcome.webp" type="image/webp" />
```

Comment:
there's a number of formatting changes in this PR like this. it'd be better to send those as a separate PR to make it easier to review. there's no reason for this file to be on this PR

Reply:
Hmm, I think this came from one of the merges from upstream.

Reply:
I don't think so: https://github.com/sveltejs/kit/blob/master/packages/create-svelte/templates/default/src/routes/index.svelte

Reply:
seems to have happened here https://github.com/sveltejs/kit/pull/1130/commits/d2ba69ac1f7b6dba8d6de6e8a4f3b363f9cecc31?file-filters%5B%5D=.svelte#diff-1b1ea3636422b9254ef3d03ed1f4acad0c7726865ffd752ec6d83d68b22a28d9R17

Reply:
but is it necessary to remove this change? seems trivial

Reply:
It was the automatic formatting on save that changed it. I removed those changes now ;-)